src/tbb/src/tbbmalloc/TypeDefinitions.h

```diff
@@ -25,7 +25,7 @@
 #       define __ARCH_ipf 1
 #   elif defined(_M_IX86)||defined(__i386__) // the latter for MinGW support
 #       define __ARCH_x86_32 1
-#   elif defined(_M_ARM)
+#   elif defined(_M_ARM) || defined(__aarch64__)
```

Comment:
Do you know if this change exists upstream as well in newer TBB versions? I worry a bit about losing TBB patches in our bundled copy if we later want to pull a newer version of TBB into RcppParallel.

Reply:
Unfortunately Windows ARM64 compatibility was only officially added in oneTBB, so there isn't an upstream reference to draw from.

But to help record the changes, I've copied the approach from `RcppEigen` and added a `patches` directory with a diff of the TBB changes